lisa/operating_system.py

```diff
@@ -504,26 +500,40 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
         return False
 
     def _get_information(self) -> OsInformation:
-        cmd_result = self._node.execute(
-            cmd="lsb_release -a", shell=True, no_error_log=True
-        )
+        # try to set version info from /etc/os-release.
```

Comment:
wait, don't we already get the version and rest of the info from /etc/os-release in Posix class as well?
Why do we need it here?

Reply:
I see. Can we somehow re-use the code snippet used in Posix class to get the os information from /etc/os-release and then just replace the version part from /etc/debian_version?

Reply:
I'll refine it later.